src/Root.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { App } from './App';
+import { Navigate, Route, Routes } from 'react-router-dom';
+import { HomePage } from './components/HomePage/HomePage';
+import { PeoplePage } from './components/PeoplePage/PeoplePage';
+import { NotFoundPage } from './components/NotFoundPage/NotFoundPage';
+
+export const Root = () => {
+  return (
+    <Routes>
+      <Route path="/" element={<App />}>
+        <Route index element={<HomePage />} />
+        <Route path="people">
+          <Route path=":slagId?" element={<PeoplePage />} />
```

Comment:
```suggestion
          <Route path=":slug?" element={<PeoplePage />} />
```
Just slug, https://developer.mozilla.org/en-US/docs/Glossary/Slug